Extensions/GoogleOptimize/Src/Tasks/GoogleOptimize/models/TaskParameter.ts

```diff
@@ -0,0 +1,75 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+
+export class TaskParameter{
+    private  endpoint :string;
+    private  accountId :string  ;
+    private  webPropertyId : string ;
+    private  profileId : string ;
+    private  experimentName: string ;
+    private  experimentId :string ;
+    private  action:string ;
+    private  trafficCoverage : string | null;
+    private  equalWeighting : string | null;
+    private  filePath: string ;
+
+    constructor() {
+        try{
+            this.endpoint = tl.getInput('googleEndpoint' , true);
```

Comment:
Second argument here means optional. Can you validate if we passed argument value correctly.